src/main/kotlin/uk/gov/justice/digital/hmpps/authorizationserver/resource/ClientDeploymentController.kt

```diff
@@ -29,6 +30,18 @@ class ClientDeploymentController(
       "username" to authenticationFacade.currentUsername!!,
       "baseClientId" to clientIdService.toBase(clientDeployment.clientId),
     )
+    telemetryClient.trackEvent("AuthorizationServerClientDeploymentDetailsAdded", telemetryMap)
+  }
+
+  @PutMapping("clients/deployment/update")
```

Comment:
I wonder if we want to change these so they're a bit more RESTful?

so `POST /clients/deployment`
and `PUT /clients/deployment/<deployment id>`

Reply:
Updated as RESTful but used the existing add request object to all update request details